hudi-common/src/main/java/org/apache/hudi/common/model/HoodieCommitMetadata.java

```diff
@@ -149,7 +149,7 @@ public List<String> getFullPathsByPartitionPath(String basePath, String partitio
     if (getPartitionToWriteStats().get(partitionPath) != null) {
       for (HoodieWriteStat stat : getPartitionToWriteStats().get(partitionPath)) {
         if ((stat.getFileId() != null)) {
-          String fullPath = FSUtils.constructAbsolutePathInHadoopPath(basePath, stat.getPath()).toString();
```

Comment:
You should delete the method in FSUtils

Reply:
This method is still used in other modules, so we cannot delete it at the moment.

Reply:
ok